ctsit/copy_email_alert_log_to_field
# Initial working module

To test:
1. Activate the module on any project which has had alert emails sent (e.g. a local clone of the FRCovid project).
1. Set **Field to pipe alert log** to an appropriate field (e.g. `result_comments`)
1. Check **Search all alerts on next run**
1. Wait at least 1 minute
1. Navigate to any record and event containing the field you specified (note the event must be used in at least one alert)
1. Observe that the comments field is now populated with "<alert_title> alert email sent: <timestamp>" lines

___

~~The **Hours back to search for alerts** option is a bit pointless in light of the **Search all alerts on next run** option. I'd like to remove it before this PR is merged unless there's a compelling argument to keep it. Also its name is bad.~~. 
Configuration removed in commit 5283f2a, now only the past 8 minutes of emails are checked (cron runs every minute, allowed to take a maximum of 6 minutes, 2 extra buffer minutes are allotted in the event of a one-off error)
